src/components/interaction-designer/block-types/Core_SetContactPropertyBlock.vue

```diff
@@ -7,22 +7,21 @@
       :uses-default-contact-props-editor="usesDefaultContactPropsEditor"
       :uses-default-branching-editor="usesDefaultBranchingEditor"
       @handleBranchingTypeChangedToUnified="handleBranchingTypeChangedToUnified({block})">
-      <template slot="extras">
-        <contact-property-editor
-          v-if="!$slots['extras']"
-          :block="block" />
-        <slot
-          v-if="$slots['extras']"
-          name="extras" />
-      </template>
-      <template slot="branching">
-        <slot
-          name="branching" />
-      </template>
-      <template slot="contact-props">
-        <slot
-          name="contact-props" />
-      </template>
+      <slot
+        slot="resource-editors"
+        name="resource-editors" />
+      <slot
+        slot="extras"
+        name="extras">
+        <contact-property-editor :block="block" />
+      </slot>
+      <slot name="vendor-extras" />
+      <slot
```

Comment:
This is interesting @bulatgab , I never knew this before. This is the equivalent of
```
<template slot="branching">
        <slot name="branching" />
</template>
```
right ? Could you share me the related doc explaining this?

Reply:
@safydy-r Yes, exactly.
I saw people using it somewhere on forums, probably at vue github issues. But I couldn't find it now, where exactly it was, unfortunately.

Reply:
@safydy-r 
Found it. I think it was this comment: https://forum.vuejs.org/t/set-grandchild-slot-content-via-parent/49995/3
See `<slot name="body" slot="body"></slot>`

Reply:
This is great! Took me a while to get it. I myself copied a stack overflow approach without considering this much better one.